x/wasm/keeper/genesis.go

```diff
@@ -55,11 +55,13 @@ func InitGenesis(ctx sdk.Context, keeper *Keeper, data types.GenesisState, staki
 	}
 
 	// sanity check seq values
-	if keeper.peekAutoIncrementID(ctx, types.KeyLastCodeID) <= maxCodeID {
-		return nil, sdkerrors.Wrapf(types.ErrInvalid, "seq %s must be greater %d ", string(types.KeyLastCodeID), maxCodeID)
+	seqVal := keeper.peekAutoIncrementID(ctx, types.KeyLastCodeID)
+	if seqVal <= maxCodeID {
+		return nil, sdkerrors.Wrapf(types.ErrInvalid, "seq %s with value: %d must be less than or equal to maxCodeID: %d ", string(types.KeyLastCodeID), seqVal, maxCodeID)
```

Comment:
The error message is not correct anymore. The sequence must be greater as defined in Line 59

Reply:
Ah, I see I have this backwards as the check determines if we throw an error... thanks!